modules/uikit/CMakeLists.txt

```diff
@@ -46,9 +46,9 @@ file(GLOB ${PROJECT_NAME}_ui
 )
 
 file(GLOB MOC_HEADERS
-    "includes/converters/*.h"
```

Comment:
What the reason of this?

Reply:
The order of the MOC Headers for UIKIT was breaking CMake locally. ui_uiedit.h was generating at the wrong time when using multiple cores to build. Ninja 1.11.1 and changing the order has locally consistently fixed the issue for me but suggestions are welcomed.